<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMSIImpedanceWidget</class>
 <widget class="QWidget" name="TMSIImpedanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="m_doubleSpinBox_manualImpedanceValueScale">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
     <property name="value">
      <double>0.000053000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="TMSIPLUGIN::TMSIImpedanceView" name="m_graphicsView_impedanceView"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="m_pushButton_start">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="m_verticalSlider_manualImpedanceValue">
     <property name="maximum">
      <number>200000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="m_pushButton_stop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="m_groupBox_Options">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="m_pushButton_takeScreenshot">
        <property name="text">
         <string>Screenshot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pushButton_loadLayout">
        <property name="text">
         <string>Load Layout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pushButton_saveValues">
        <property name="text">
         <string>Save Values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_pushButton_Help">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TMSIPLUGIN::TMSIImpedanceView</class>
   <extends>QGraphicsView</extends>
   <header>tmsiimpedanceview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
